src/main/java/amazon/AllSearchResultPage.java

```diff
@@ -33,10 +35,16 @@ public boolean textVisibilityWhenClickOnBestSellerButton() {
         Random random=new Random();
         int randomNumber=random.nextInt(1, listItemWhereWriteBestSeller.size());
         Actions act=new Actions(driver);
-        act.moveToElement(listItemWhereWriteBestSeller.get(randomNumber)).build().perform();
-        boolean textVisibility = textListWhenClickOnBestSellerButton.get(randomNumber).getText().contains("in");
+        boolean textVisibility=false;
+       try {
+           act.moveToElement(listItemWhereWriteBestSeller.get(randomNumber)).build().perform();
+           textVisibility = textListWhenClickOnBestSellerButton.get(randomNumber).getText().contains("in");
+       }catch (NoSuchElementException e){
```

Comment:
For example if best seller item text doesn't contain "in", test should fail? From where do you know such a requirement? Read carefully the requirement, then write a code.